### DISCUSSION TITLE
100% CPU

### DISCUSSION THREAD

**Discussion by lucmp** [Motion-Project/motion#1426] [General]
I installed motion on my desktop to test it and learn to use it, and I really liked it. Besides its expected features, I noticed that it never went up past 20% of CPU use. I found that really excellent.

When I was finally ready, I installed it on a small netbook that I keep running continuously to monitor my camera stream. However, motion never drops below 80% use on my netbook, averaging 100% and often reaching 130%.

My desktop is 64bit and the netbook is 32bit. Is that the reason they perform so differently?

**Comment by Mr-Dave** [comment]
I wouldn't think it would be the 64 vs 32 but instead the CPU.  If you want you could check the difference in CPU benchmarks and I'd guess any recent desktop is going to be well over 7x faster than a netbook.

**Comment by tosiara** [comment]
It is normal to see high CPU on a low power or low end devices. What are the specs of your netbook? Which CPU it is running?

**Comment by lucmp** [comment]
It's an Asus EEE PC. Intel Atom CPU N270 @ 1.60GHz.
